manual/integrate_root_into_my_cmake_project/index.md

```diff
@@ -55,36 +60,33 @@ Generate dictionary for a set of header files (convenient wrapper on top of the
       [OPTIONS opt1 opt2 ...] )
 {% endhighlight %}
 
-The `${dictionary}.cxx` and `${dictionary}.pcm` files are created using the headers and the selection file, calling the `genreflex` command. Optionally, you can provide `OPTIONS` to customize the way the `genreflex` command is called.
-
 ## Full example (event project)
 
 The following is an example of a project that creates a library including a dictionary and an executable file.
 
 {% highlight C++ %}
-# CMakeLists.txt for event package. It creates a library with a dictionary and a main program.
+# CMakeLists.txt for the "event" package. It creates a library with a dictionary and a main program.
+# If ROOT is not installed in a default system location you need to tell CMake where to find it.
+# Sourcing `thisroot.sh` already sets the required environment variables.
+# Otherwise, you'll have to tell cmake where to look for ROOT,
+# e.g. by passing `-DCMAKE_PREFIX_PATH="/path/to/root/installation` at configuration time
+
 cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
 project(event)
 
-# You need to tell CMake where to find the ROOT installation. This can be done in a number of ways:
-#   - ROOT built with classic configure/make use the provided $ROOTSYS/etc/cmake/FindROOT.cmake.
-#   - ROOT built with CMake. Add in CMAKE_PREFIX_PATH the installation prefix for ROOT.
-list(APPEND CMAKE_PREFIX_PATH $ENV{ROOTSYS})
-
-# Locates the ROOT package and defines a number of variables (e.g. ROOT_INCLUDE_DIRS).
+# Locate the ROOT package and define a number of useful targets and variables.
 find_package(ROOT REQUIRED COMPONENTS RIO Net)
```

Comment:
Let's add here as well `Tree` to make it match to list of `target_link_libraries`?

Reply:
or remove `ROOT::Tree` from the list above, I don't know why I added it

Reply:
(done)